### PR TITLE
Javascript/javascript1/week1

### PR DESCRIPTION
**Pull request by Jul-S**

### CODE REVIEW

File: javascript/javascript1/week1/ez_namey.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const firstWords = ["Easy", "Awesome", "Corporate", "Severe", "Smiling", 
+    "Heavenly", "Childlike", "Shy", "Anxious", "Ugliest", "Accurate"];
+const secondWords = ["World", "Ladder", "Exam", "Driver", "Organization",
+    "Operation", "Sector", "Oven", "Atmosphere", "Product"];
+
+const randomNumber = Math.floor(Math.random() * 10) + 0;
```
Comment (FrederikNygart) [inline javascript/javascript1/week1/ez_namey.js]: Is '+ 0' needed?

File: javascript/javascript1/week1/ez_namey.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const firstWords = ["Easy", "Awesome", "Corporate", "Severe", "Smiling", 
+    "Heavenly", "Childlike", "Shy", "Anxious", "Ugliest", "Accurate"];
```
Comment (FrederikNygart) [inline javascript/javascript1/week1/ez_namey.js:2]: Minor inconsequential 'bug' the array has 11 items, ranging from index 0-10. The highest number generated at random is 9, therefore index 10(Accurate) will never he hit.

As a fun small side quest, you could consider making the random number generator depend on the length of your array. Thereby ensuring that all items get a chance to be featured, and make it robust if in the future you want to change the number of items in the array ;)

Let me know if you need any help with this.

File: javascript/javascript1/week1/goodboy-oldboy.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+
+let dogYear = dogYearFuture - dogYearOfBirth;
+
+// 1st case showing in human years
+let shouldShowResultInDogYears = false;
```
Comment (FrederikNygart) [inline javascript/javascript1/week1/goodboy-oldboy.js:6]: You have to use 'conditionals' to either run one or the other command.

quick intro to the syntax of conditionals:
```js
if (condition){
// execute this code
}else{
 // execute some other code
}
```

File: javascript/javascript1/week1/housey_pricey.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+//Peter`s case:
+let width = 8;
+let depth = 10;
+let heght = 10;
```
Comment (FrederikNygart) [inline javascript/javascript1/week1/housey_pricey.js]: height - spelling correctly is crucial in large distributed systems in order to collaborate and understand your teammates 🥇.

File: javascript/javascript1/week1/housey_pricey.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+let gardenSizeInM2 = 100;
+let paidPrice = 2_500_000;
+
+//First we calculte house volume:
+let volumeInMeters = depth * heght * width;
+// then use it in house price formula:
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+console.log(`Peter's house price should be ${housePrice}, and he paid ${paidPrice}`);
+
```
Comment (FrederikNygart) [inline javascript/javascript1/week1/housey_pricey.js:21]: Minor 'extra' assignment: Can you also log whether the person, should actually buy the house?

eg. 'Peter should buy the house: @{shouldBuyTheHouse}'

File: javascript/javascript1/week1/pizza.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+// 1st Case its not a family pizza
+let familySize = false;
+let totalPrice = pizzaPrice * amountOfPizza;
+console.log(`New pizza order: ${amountOfPizza} ${favPizza}. The price of the pizza is: ${totalPrice}`);
+
+// 2nd case it`s a family pizza
+familySize = true; 
+totalPrice *= 2;
+console.log(`New pizza order: ${amountOfPizza} family ${favPizza}. The price of the pizza is: ${totalPrice}`);
```
Comment (FrederikNygart) [inline javascript/javascript1/week1/pizza.js]: Apply conditionals as stated in previous comment 'Goodboy-Oldboy'

**Review by FrederikNygart — CHANGES_REQUESTED** [review]
Really good job! have understood many important concepts such as variables, arrays, boolean values.

However, there are still some comments, so give a try, and if you need help feel free to write me on slack 🤙

File: javascript/javascript1/week1/ez_namey.js (hunk 1 above)
Comment (Jul-S) [reply javascript/javascript1/week1/ez_namey.js]: Not really :)  just copy-pasted from homework insructions didn`t check it

File: javascript/javascript1/week1/ez_namey.js (hunk 2 above)
Comment (Jul-S) [reply javascript/javascript1/week1/ez_namey.js:2]: Wow! Nice catch! Fixed it in next commit

File: javascript/javascript1/week1/goodboy-oldboy.js (hunk 3 above)
Comment (Jul-S) [reply javascript/javascript1/week1/goodboy-oldboy.js:6]: Thanks! Fixed it!

File: javascript/javascript1/week1/pizza.js (hunk 6 above)
Comment (Jul-S) [reply javascript/javascript1/week1/pizza.js]: Commit added

File: javascript/javascript1/week1/housey_pricey.js (hunk 4 above)
Comment (Jul-S) [reply javascript/javascript1/week1/housey_pricey.js]: Got it! Thank you!

File: javascript/javascript1/week1/housey_pricey.js (hunk 5 above)
Comment (Jul-S) [reply javascript/javascript1/week1/housey_pricey.js:21]: Added extra boolean to show 'Peter should buy the house: @{shouldBuyTheHouse}'

**Review by FrederikNygart — APPROVED** [review]
Excellent job 🚀

**Pull request merged**